app/controllers/dynamic_forms/api/form_controller.rb

```diff
@@ -10,10 +10,19 @@ class FormController < DynamicForms::Api::BaseController
       before_action :validations
       before_action :remove_unnecessary_params, only: [:create]
 
+      def show
+        unless @custom_form.json_schema_available?
+          return render json: { message: "JSON-Schema is not enabled"},
+            status: :unprocessable_entity
+        end
+        render json: @custom_form
+      end
```

Comment:
I think you can use an if else to be more clear

if @custom_form.json_schema_available?
 render json: @custom_form
else
  return render json: { message: "JSON-Schema is not enabled"},
            status: :unprocessable_entity
end


Maybe it is more legible, what do you think? @Alejando